src/module/entities/validator/v1.js

```diff
@@ -14,7 +14,7 @@ module.exports = (req) => {
 			req.checkBody('name').exists().withMessage('required name ')
 		},
 		update: function () {
-			req.checkBody('externalId').exists().withMessage('required externalId')
+			req.checkBody('metaInformation.externalId').exists().withMessage('required externalId')
```

Comment:
@MallanagoudaB why we have to make sure metaInformation.externalId is passed?

Reply:
That the same issue we connect before.

Reply:
@MallanagoudaB what we if pass metaInformation: {}